docs/source/models/cause.rst

```diff
@@ -255,13 +254,13 @@ and discussed in more detail afterward.
      - Rate at which an individual accrues years lived with disability due to
        the state in the cause model.
    * - `Cause-specific Mortality`_
-     -
-     -
-     -
+     - Mortality rate from a specified cause for a population
```

Comment:
We want to describe the numerator and denominator.  E.g. Deaths due to a specific cause per person-year for the entire population.